src/conversions/std/array.rs

```diff
@@ -36,6 +38,40 @@ where
     }
 }
 
+impl<'py, T, const N: usize> IntoPyObject<'py> for [T; N]
+where
+    T: IntoPyObject<'py>,
+{
+    type Target = PyList;
```

Comment:
I guess there's a relevant discussion point here if `[u8; N]` should be treated as a byte slice and become `PyBytes` (in which case this here might be `PyAny`, unless we can find a way to do a type-level specialization on this associated type).